src/main/java/com/example/inventoryservice/dto/UserDto.java

```diff
@@ -0,0 +1,37 @@
+package com.example.inventoryservice.dto;
+
+import com.example.inventoryservice.entity.Restaurant;
+import com.example.inventoryservice.entity.Role;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class UserDto {
+
+    private Long id;
+
+    @NotBlank(message = "First name can't be empty.")
+    private String firstName;
+
+    @NotBlank(message = "Last name can't be empty.")
+    private String lastName;
+
+    private List<Role> userRole = new ArrayList<>();
+
+    @NotBlank(message = "Username can't be empty.")
+    private String username;
+
+    @NotBlank(message = "Password can't be empty.")
+    private String password;
```

Comment:
[major] There is no need in returning the password to the API client. We don't want to show it anywhere.